File: domain/src/main/java/com/study/platform/domain/post/model/PostSearchResult.java
package com.study.platform.domain.post.model;

import java.time.LocalDateTime;
import java.util.UUID;

public record PostSearchResult(
        UUID id,
        String authorNickname,
        String title,
        String techStack,
        int maxMembers,
        LocalDateTime deadline,
        StudyPostStatus status,
        LocalDateTime createdAt
) {
}

File: domain/src/main/kotlin/com/study/platform/domain/post/model/PostSearchResult.kt
package com.study.platform.domain.post.model

import java.time.LocalDateTime
import java.util.UUID

data class PostSearchResult(
    val id: UUID,
    val authorNickname: String,
    val title: String,
    val techStack: String?,
    val maxMembers: Int,
    val deadline: LocalDateTime?,
    val status: StudyPostStatus,
    val createdAt: LocalDateTime
)
